### PR TITLE
com.utilities.buildpipeline 1.3.3

### PR DESCRIPTION
**Pull request by StephenHodgson**
- added additional command line args `-development`, `-autoConnectProfiler`, and `-buildWithDeepProfilingSupport`
- replaced `-disableDebugging` with `-allowDebugging` which can either be `true` or `false`

### CODE REVIEW

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs [hunk 1]
```diff
@@ -239,6 +243,24 @@ public virtual void ParseCommandLineArgs()
                         break;
                     case "-disableDebugging":
                         EditorUserBuildSettings.allowDebugging = false;
+                        Debug.LogWarning("This arg has been deprecated. use \"-allowDebugging false\" instead.");
+                        break;
+                    case "-allowDebugging":
+                        var value = arguments[++i];
```
Comment (afarchy) [inline Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs:249]: seems odd this isn't a switch

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs [hunk 2]
```diff
@@ -206,6 +206,10 @@ public virtual void ParseCommandLineArgs()
                     case "-acceptExternalModificationsToPlayer":
                         BuildOptions = BuildOptions.SetFlag(BuildOptions.AcceptExternalModificationsToPlayer);
                         break;
+                    case "-development":
+                        EditorUserBuildSettings.development = true;
```
Comment (afarchy) [inline Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs:210]: does this do something different from the next line?

**Review by afarchy — APPROVED** [review]

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs (hunk 1 above)
Comment (StephenHodgson) [reply Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs:249]: it was before, but the logic was inverted. I figured being more explicit and deliberate was better for replacing this command

File: Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs (hunk 2 above)
Comment (StephenHodgson) [reply Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs:210]: prob the same thing, but left it in to be safer than sorry.

**Pull request merged**